src/classes/MTCH_FindGifts_CTRL.cls

```diff
@@ -27,13 +27,13 @@
     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
     POSSIBILITY OF SUCH DAMAGE.
 */
-/*
+/**
```

Comment:
Interestingly, ST/MM gives me context highlighting (inherited from JavaDoc, I presume) with an opening `/**` but gives it regular comment styling with more *s.